src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -163,6 +163,16 @@ define_themes! { [light:0, dark:1]
         }
         searcher {
             action_list_gap = 5.0, 5.0;
+            searcher = 5.0, 5.0;
+            selection {
+                padding.horizontal = 5.0, 5.0;
+            }
+        }
+        documentation {
+            background = Rgba(0.984,0.992,1.0,1.0) , Lcha(0.2,0.014,0.18,1.0);
```

Comment:
do not hardcore these values. Link them to node background. Just instead f `Rgba(...)` write `graph_editor::node::background` here.